sdcm/utils/remotewebbrowser.py

```diff
@@ -46,7 +47,15 @@ def web_driver_docker_client(self) -> Optional[DockerClient]:
         SSHAgent.add_keys((self.ssh_login_info["key_file"], ))
         # since a bug in docker package https://github.com/docker-library/python/issues/517 that need to explicitly
         # pass down the port for supporting ipv6
-        return DockerClient(base_url=f"ssh://{self.ssh_login_info['user']}@{normalize_ipv6_url(self.ssh_login_info['hostname'])}:22",
+        try:
+            return DockerClient(base_url=f"ssh://{self.ssh_login_info['user']}@{normalize_ipv6_url(self.ssh_login_info['hostname'])}:22",
+                            timeout=DOCKER_API_CALL_TIMEOUT)
```

Comment:
Pre-commit is not happy with this indentation.

Reply:
fixed